### PR TITLE
Upload .travis.yml

### PR DESCRIPTION
**Pull request by geekayush**

### CODE REVIEW

**Comment by geekayush** [comment]
@yashasingh Please review this PR in reference to #7

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+script:
+  - yarn test
+  - yarn build
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $github_token
+  local_dir: build
+  on:
+    branch: master
```
Comment (yashasingh) [inline .travis.yml]: Add new line here.

**Comment by yashasingh** [comment]
Use github keywords in PR description in order to automatically close the issue when PR merges. Here, use ```closes #7``` . Refer [link](https://help.github.com/articles/closing-issues-using-keywords/)

File: package.json [hunk 2]
```diff
@@ -17,5 +17,6 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
-  }
+  },
+  "homepage": "https://geekayush.github.io/ncs-jss/Proj_kc01/"
```
Comment (geekayush) [inline package.json]: @yashasingh  Is this homepage attribute fine? I am not sure with this.

Comment (geekayush) [reply package.json]: Should it be: https://ncs-jss.github.io/Proj_kc01/ ?

Comment (itaditya) [reply package.json]: @geekayush just do `npm init` in your root directory, The necessary fields in package.json will be automatically updated.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+  - "stable"
+cache:
+  directories:
+  - node_modules
+script:
```
Comment (yashasingh) [inline .travis.yml]: Use some linting tool as well. Refer [this](https://www.sitepoint.com/comparison-javascript-linting-tools/) for comparing different tools. For setting it up on Travis CI, refer [this](https://github.com/dwyl/learn-travis/blob/master/README.md)

Comment (geekayush) [reply .travis.yml]: I'll set up ESlint in a separate PR

Comment (yashasingh) [reply .travis.yml]: Alright, I'm creating an issue for setting up ESLint, resolve that :+1:

**Pull request merged**